src/domain/session/room/timeline/tiles/TextTile.js

```diff
@@ -15,15 +15,16 @@ limitations under the License.
 */
 
 import {MessageTile} from "./MessageTile.js";
+import { Linkifier } from "../../../../Linkifier.js";
 
 export class TextTile extends MessageTile {
-    get text() {
+    get messageFormat() {
         const content = this._getContent();
-        const body = content && content.body;
+        let body = content && content.body;
```

Comment:
I think the linkifier won't work if body is undefined here, so perhaps we can fall back to an empty string
```suggestion
        let body = content?.body || "";
```